client/me/memberships/subscription.jsx

```diff
@@ -135,13 +136,14 @@ function Subscription( { translate, subscription, moment, stoppingStatus } ) {
 					</Card>
 					<CompactCard
 						tagName="button"
-						className="memberships__subscription-remove"
+						className="remove-purchase__card"
 						onClick={ stopSubscription }
 					>
+						<MaterialIcon icon="delete" className="card__icon" />
 						{ isProduct
-							? translate( 'Remove %s product.', { args: subscription.title } )
```

Comment:
When this is merged, we should notify the i18n folks to see if they can somehow alter the existing translations to remove these punctuation marks without having to re-translate the string into every locale.